### DISCUSSION TITLE
Rigas satiksme (public transportation) integration

### DISCUSSION THREAD

**Discussion by flusher998** [organicmaps/organicmaps#3931] [Ideas]
When I want to use public transport I have to use site developed by rigas satiksme : https://saraksti.rigassatiksme.lv/index.html#plan/3011/1541,1542 and mostly it is unconvenient to do. As I have experienced, most of stop places are already indexed in the map (some of them are misplaced, mby old database) and could be used with data from site I provided.

**Comment by i-ky** [comment]
Rigas Satiksme [publishes data in GTFS format](https://www.rigassatiksme.lv/en/about-us/publishable-information/open-data/) and there is a [document describing GTFS use in Organic Maps](https://github.com/organicmaps/organicmaps/blob/master/docs/EXPERIMENTAL_PUBLIC_TRANSPORT_SUPPORT.md). See also #837.
